src/index.js

```diff
@@ -1,26 +1,51 @@
 import React from 'react';
-import ReactDOM from 'react-dom';
-import faker from 'faker';
+import { View, Text, Animated, Dimensions} from 'react-native';
+const screenWidth = Dimensions.get('window').width;
 
-const App = () => {
-    return(
-        <div className="ui container comments"> 
-            <div className="comment">
-                <a href="/" className="avatar">
-                    <img alt="avatar" src={faker.image.avatar()}></img>
-                </a>
-                <div className="content">
-                    <a href="/.."className="author"> Sam </a>
-                    <div className="metadata">
-                        <span className="date"> Today at 5:23pm</span>
-                    </div>
-                    <div className="text" > haha</div>
+export default class animatedView extends React.Component {
```

Comment:
There is a convention in JS and React to start name of the Class or Component with a capital letter, e.g. `AnimatedView`.
See note [here](https://reactjs.org/docs/components-and-props.html)